services/audit/pkg/config/config.go

```diff
@@ -26,7 +26,6 @@ type Config struct {
 type Events struct {
 	Endpoint             string `yaml:"endpoint" env:"OCIS_EVENTS_ENDPOINT;AUDIT_EVENTS_ENDPOINT" desc:"The address of the event system. The event system is the message queuing service. It is used as message broker for the microservice architecture."`
 	Cluster              string `yaml:"cluster" env:"OCIS_EVENTS_CLUSTER;AUDIT_EVENTS_CLUSTER" desc:"The clusterID of the event system. The event system is the message queuing service. It is used as message broker for the microservice architecture. Mandatory when using NATS as event system."`
-	ConsumerGroup        string `yaml:"group" env:"AUDIT_EVENTS_GROUP" desc:"The consumergroup of the service. One group will only get one copy of an event."`
```

Comment:
Removed but no deprecation?

Reply:
Yes you have a point. We should not remove envvars without deprecation. But in this case, since this envvar should never have been created in the first place, I would go the short way and just remove it nice and quiet.